src/actions/registerUserActions.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { REGISTER_USER_SUCCESS, REGISTER_USER_ERROR } from './types';
+
+export const fetchUsers = (postData) => dispatch => {
+  axios
+    .post('http://127.0.0.1:8000/api/users/', postData)
+    .then((response) => {
+      localStorage.setItem('auth_token', response.data.user.auth_token);
+      dispatch({ type: REGISTER_USER_SUCCESS, payload: true });
+      toast.success('Signup successful', { autoClose: false });
+    })
+    .catch((error) => {
+      let emailKey = 'Email: ';
```

Comment:
Please use a for-in loop to check the errors object. [Link](https://stackoverflow.com/questions/684672/how-do-i-loop-through-or-enumerate-a-javascript-object)